northstar/src/api/model.rs

```diff
@@ -224,7 +226,7 @@ pub enum UmountResult {
 pub enum Response {
     Ok,
     Error(Error),
-    ContainerStats(Container, ContainerStats),
+    Inspect(Box<ContainerData>),
```

Comment:
Why is the `Box` needed here? Because of the huge size difference between this variant and the others?

Reply:
exactly :)

Reply:
The `Box` is transparent in json. Right?

Reply:
right